lib/Workflow/ActionsButtonService.php

```diff
@@ -55,6 +55,7 @@ public function getAllowedTransitions(Workflow $workflow, ElementInterface $elem
                 'iconCls' => $transition->getIconClass(),
                 'objectLayout' => $transition->getObjectLayout(),
                 'notes' => $notes,
+                'saveBehavior' => $transition->getOptions()['saveBehavior']
```

Comment:
```suggestion
                'saveBehavior' => $transition->getOptions()['unsavedChangesBehaviour']
```
Here it should be something like this, right? And I think it would be good if we spell "behaviour" always in the same way, either "behaviour" or "behavior". What do you think?

Reply:
You are right. Have fixed naming to "behaviour" in all the different places.